.travis.yml

```diff
@@ -8,20 +8,7 @@ services:
 cache:
   directories:
     - $HOME/.m2
-    - $HOME/.cache
-    - $HOME/google-cloud-sdk
-
-before_install:
-  - docker pull akvo/flow-maven-build
-  - if [[ ! -f "$HOME/google-cloud-sdk/bin/gcloud" ]]; then rm -rf "$HOME/google-cloud-sdk";
-    curl https://sdk.cloud.google.com | bash > /dev/null; fi
-  - pip install pyopenssl
 
 script:
+  - docker build -t akvo/flow-maven-build .
```

Comment:
As we already have a akvo/flow-maven-build image somewhere in the internet, we should change the name again so that we do not confuse the now local build image with the global one.